src/download_games/read_chesscom.py

```diff
@@ -3,18 +3,18 @@
 # This program works as of 5/05/2020
 
 
-import glob
-import os
-import urllib.request
-from ast import literal_eval
-from pathlib import Path
+import glob # pragma: no cover
```

Comment:
Are these pragmas really required?

Reply:
They are included so that the coverage ignores those lines.
For now yes, when I have tests for this file I will be able to remove them.

Reply:
Add coverage file to exclude import lines.